### DISCUSSION TITLE
Low correlation between PsPM and Ledalab

### DISCUSSION THREAD

**Discussion by TeaforTwo1011** [bachlab/PsPM#454] [Q&A] [answered]
Hi Dominik and all,

I hope the question is not offensive, because I would like to do a comparison of different preprocessing pipelines on SCR (peaking-scoring, Ledalab, and PsPM(DCM)) with my own dataset. 

What I am interested in is the anticipatory SCR amplitude, and I followed your previous suggestion, to model it in DCM. I exported the statistics of DCMs, taking the flexible responses amplitude as the anticipatory SCR.

After days of running, I finally got all my results, but surprisingly, I found that the correlation between PsPM and Ledalab results were only 0.28. I am not sure if this is a reasonable result or not. 

Could you maybe give me some suggestions?

Thank you very much!!

**Comment by dominikbach** [comment, marked as answer]
Hi there

always good to see PsPM in competition!

In general, whether the correlation between two methods is even meaningful, and if yes then whether it should be high, depends on what you are actually correlating, in this case e.g. individual trials within a condition, across conditions, or condition averages; and what the conditions are.

(Explanation: correlation between methods can be a test of convergent validity and thus meaningful if you expect variability in the true scores - however if this is the case and one of the methods is bad and the other is good then you might expect a low correlation. If you don't expect variability in the true scores in the first place, then you are assessing noise correlations, which have a rather different meaning). 

Could you give more details?

Dominik
